ou_dedetai/installer.py

```diff
@@ -289,6 +290,14 @@ def ensure_product_installed(app: App):
             process.wait()
 
     # Clean up temp files, etc.
+    mst_destination = Path(app.conf.install_dir) / "data/wine64_bottle/drive_c/LogosStubFailOK.mst"
```

Comment:
Why was this moved here?

The place in the code that a temporary file is created should be the place where it's cleaned up